### ISSUE TITLE
Not all unused variables detected

### ISSUE THREAD

**Issue by Koc** [kalessil/phpinspectionsea#856]
```php
        if (null !== $key = $this->options['routing_field']) {
            $this->index->deleteByQuery((new BoolQuery())->addFilter(new Terms('id', $ids)));

            return;
        }
```

variable `$key` not detected as unused.

**Comment by kalessil** [comment]
Done, gets reported now
